preupg/xmlgen/compose.py

```diff
@@ -96,11 +132,11 @@ def collect_group_xmls(source_dir, content=None, level=0,
                 directories = [x for x in os.listdir(new_dir)
                                if not os.path.isdir(os.path.join(new_dir, x))]
                 if not directories and 'postupgrade.d' not in dirname:
-                    print ("WARNING: It seems that group.ini file is missing"
-                           " in %s. Please check if it is really missing."
-                           % new_dir)
+                    print("WARNING: It seems that group.ini file is missing"
+                          " in %s. Please check if it is really missing."
+                          % new_dir)
```

Comment:
Simplify the warning to: 
`"group.ini file is missing in %s."`
and use _log_message_ function for it, with _level=logging.WARNING_.